lib/puppet/provider/firewall/ip6tables.rb

```diff
@@ -158,6 +168,7 @@ def self.iptables_save(*args)
     :addrtype  => [:src_type, :dst_type],
     :iprange   => [:src_range, :dst_range],
     :owner     => [:uid, :gid],
+    :time      => [:time_start, :time_stop, :month_days, :week_days, :date_start, :date_stop, :time_contiguous, :kernel_timezone]
```

Comment:
in case time ever changes, with this many parameters, it might be more convenient to have them in a row, rather than a line…